Bibliotheque nationale de France.js

```diff
@@ -499,9 +469,11 @@ var BnfClass = function() {
 	*/
 	this.getDCType = function(doc, url) {
 		try {
-			var xPath = '//div[@class="notice-detail"]//div[@id="type"]';
+			// var xPath = '//div[@class="notice-detail"]//div[@id="type"]';
+			var xPath= './/*[@id="corps"]/div[1]/div[3]/div/h1';
```

Comment:
this seems very fragile. No way to identify this with an attribute or leave out the `div[1]/div[3]`?